juicy-draggable.html

```diff
@@ -252,7 +252,13 @@
             var element = null;
             var handle = null;
             var that = this;
-            var root = this.createShadowRoot();
+            var root;
+            if (this.attachShadow) {
+                root = this.attachShadow({ mode: 'open' });
+            } else {
+                root = this.createShadowRoot();
```

Comment:
This will create shadowRoot, but V0 so the `<slot>` will not work. to make it work you need to use `<content>`.

I'd suggest to reemove `<template>` from line no 8, and create this inline here:
```js
if (this.attachShadow) {
    root = this.attachShadow({ mode: 'open' });
    root.innerHTML = '<slot id="content"></slot>';
} else {
    root = this.createShadowRoot();
    root.innerHTML = '<content id="content"></content>';
}
this.refreshAttributeValues();
// ...
```